checkstyle-style.xml

```diff
@@ -273,7 +273,7 @@
     <module name="RegexpHeader">
         <property name="severity" value="error" />
         <property name="header"
-            value="// Copyright 20[0-1][0-9],? (Yahoo Inc.|Oath Inc.)\n// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms\.\n" />
+            value="// Copyright 20[0-1][0-9],? (Yahoo Inc.|Oath Inc.|Verizon Media Inc.)\n// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms\.\n" />
```

Comment:
Sadly we need to revert this.  It turns out we're not allowed to use the Verizon Media in copyright's yet.